docs/samples/fluid/README.md

```diff
@@ -0,0 +1,337 @@
+
+We will use the following [machine types](https://aws.amazon.com/ec2/instance-types/m5/) for our tests:
+
+* Data: m5.xlarge
+* Workload: m5.2xlarge, m5.4xlarge
+
+We will use the following Fluid runtimes for our tests:
+
+* JindoFS
+
+For the purposes of our benchmark, we will assume that nodes are readily available and that images are cached in the nodes, so we will not be factoring in node provisioning time or image pulling time.
+
+Other assumptions:
+
+* S3 bucket and Cluster are in the same region (eu-central-1)
+* Data are preloaded in workers
+
+
+Command to measure the total time:
+
+```sh
+# Total time includes: pod initialization and running + download model (storage initializer) + load model + inference + network
+curl --connect-timeout 3600 --max-time 3600 -o /dev/null -s -w 'Total: %{time_total}s\n' -H "Content-Type: application/json" -H "Host: ${SERVICE_HOSTNAME}" "http://${INGRESS_HOST}:${INGRESS_PORT}/v1/models/bloom:predict" -d '{"prompt": "It was a dark and stormy night", "result_length": 50}'
+# Total time: 53.037210s, status code: 200
+```
+
+| Model Name                                                                | Model Size (Snapshot) | Machine Type | KServe + Storage Initializer                         | KServe + Fluid(JindoFS)                    |
+|---------------------------------------------------------------------------|-----------------------|--------------|------------------------------------------------------|--------------------------------------------|
+| [ bigscience/bloom-560m ]( https://huggingface.co/bigscience/bloom-560m ) | 3.14GB                | m5.2xlarge   | total: 52.725s (download: 25.091s, load: 3.549s)     | total: 23.286s (load: 4.763s) (2 workers)  |
+| [ bigscience/bloom-7b1 ]( https://huggingface.co/bigscience/bloom-7b1 )   | 26.35GB               | m5.4xlarge   | total: 365.479s (download: 219.844s, load: 102.299s) | total: 53.037s (load: 24.137s) (3 workers) |
+
+> NOTE: `total` is the response time of sending an inference request to a serverless inference service (scale from 0), which includes pod startup, `model download`, `model load`, model inference and network time.
+> TBD: add testing results for other runtimes
+
+As can be observed, Fluid has demonstrated notable improvements in the autoscaling performance of KServe on LLM.
+
+### Limitations
+
+One potential limitation of this integration is the use of an additional Fuse-sidecar, which may lead to increased resource consumption. However, in `the use case of LLMs`, where the instance size is typically very large and the pod consumes the entire instance, the resource usage of the Fuse-sidecar is expected to be negligible. Nonetheless, it is important to carefully consider the resource impact of the Fuse-sidecar in your specific use case to ensure that it remains within acceptable limits.
```

Comment:
KServe also has model agent sidecar which can act as model puller, I think it is possible to integrate model agent with fluid so we do not need another `fuse-sidecar`? @lizzzcai @cheyang

Reply:
The fluid removes the download step in the storage initializer:

For KServe + Storage initiazlier:
- storage initializer downloads model to /mnt/models
- application loads model from /mnt/models

For KServe + Fluid:
- application loads model from /mnt/models (model is mounted by PVC, storage initializer is not needed any more)

To integrate with model-agent (which is able to download models dynamically), one possible approach is:
- model agent `dynamically` mounts object storage (for example: `model-1`) provided by user ( for example calling the `alluxio fs mount /mnt/model-1 s3://<S3_BUCKET>/<S3_DIRECTORY>/model-1`)
- model agent symlink the mount point (`/mnt/model-1`) above to `/mnt/models/model-1`, through a shared volume.

I feel like it is possible, maybe @cheyang can comment on it. However, this will need extra development works, we can put it into the next step. I am planning to restructure the folder so that future different use case can be added under `docs/samples/fluid`, the current example will be put under `docs/samples/fluid/sidecar`. @yuzisun

Reply:
It is definitely out of scope here but would like to explore a better kserve/fluid integration as next step. I will create an issue for this.